jobs/release.Jenkinsfile

```diff
@@ -78,10 +90,12 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
                 --arch=all --url=s3://${s3_stream_dir}/builds \
                 --aws-config-file \${AWS_BUILD_UPLOAD_CONFIG}
             """)
-            }
```

Comment:
This needs to be in the other AWS creds PR.

Reply:
moved!